src/test/TestHelper.h

```diff
@@ -77,11 +77,11 @@ class TestAdaptiveStream : public adaptive::AdaptiveStream
   virtual bool Download(const DownloadInfo& downloadInfo, std::vector<uint8_t>& data) override;
 };
 
-class AESDecrypter : public IAESDecrypter
+class AESDecrypterTest : public IAESDecrypter
 {
 public:
-  AESDecrypter(const std::string& licenseKey) : m_licenseKey(licenseKey){};
-  virtual ~AESDecrypter() = default;
+  AESDecrypterTest(const std::string& licenseKey) : m_licenseKey(licenseKey) {};
+  virtual ~AESDecrypterTest() = default;
 
   void decrypt(const AP4_UI08* aes_key,
```

Comment:
on the line 96 (i cant comment below)
please comment the method
`// bool RenewLicense(const std::string& pluginUrl);`
an oversight of what was explained on the my previous comment